connectMeteorRedux.js

```diff
@@ -19,17 +19,28 @@ const meteorReduxReducers = (
       return { ...state, userId: id };
     }
     case 'RECENTLY_ADDED': {
-      return {
-        ...state,
-        reactNativeMeteorOfflineRecentlyAdded: [
-          ...(state.reactNativeMeteorOfflineRecentlyAdded || []),
```

Comment:
Is there a reason this wasn't working?

Reply:
I certainly had the case it was undefined or empty and it didn't work as expected as described in #58